packages/fluentui/react-northstar/test/specs/commonTests/isConformant.tsx

```diff
@@ -65,6 +66,8 @@ export default function isConformant(
   const { throwError } = helpers('isConformant', Component);
 
   const componentType = typeof Component;
+  // composed components store `handledProps` under config
+  const handledProps = Component.handledProps || (Component as ComposedComponent).fluentComposeConfig?.handledProps;
```

Comment:
Shuoldn't this be:
```
const handledProps = (Component as ComposedComponent).fluentComposeConfig?.handledProps || Component.handledProps;
```

Reply:
Hm... Good point. Actually these is no difference as there is no `handledProps` in static props